src/views/Home/components/FAQ/faq.scss

```diff
@@ -1,3 +1,106 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
+@import url("https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap");
```

Comment:
Not entirely sure why we'd need this? Don't we already have these fonts in from the schedule component?